apps/projects/app/components/Content/Settings.js

```diff
@@ -421,28 +437,24 @@ Settings.propTypes = {
   onLogin: PropTypes.func.isRequired,
 }
 
-const StyledInputDropDown = ({ children }) => {
+const StyledInputDropDown = ({ children, className }) => {
   const theme = useTheme()
 
   return (
-    <div css={`
+    <div className={className} css={`
       display: flex;
-      width: 240px;
       > :first-child {
         border-radius: 3px 0 0 3px;
         border: 1px solid ${theme.border};
         box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.03);
-        width: 140px;
+        flex: 1;
         z-index: 1;
-        :focus {
-          outline: 0;
-          border: 1px solid ${theme.infoSurface};
+        &, &:focus-within {
```

Comment:
I would like to ask what is doing this new rule introduced here? I tried to remove it on my browser and didn't notice any change to the UI.
I also checked https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within but is still not clear how it applies to our current use case.

Reply:
Make sure you click into the input. We are adding border to the div, not the input itself. `:focus` applies directly to an input; `:focus-within` allows adding border to a div when an element inside it has focus.